gson/src/main/java/com/google/gson/ReflectionAccessFilter.java

```diff
@@ -117,6 +117,11 @@ public FilterResult check(Class<?> rawClass) {
               ? FilterResult.BLOCK_INACCESSIBLE
               : FilterResult.INDECISIVE;
         }
+
+        @Override
```

Comment:
Can you mention these in the PR description?

Reply:
Have included it now; sorry that I added this in a follow-up commit and not in the original changes of this PR. I just remembered that this change was something I wanted to do for quite some time because the default `toString()` during debugging makes it difficult to understand which filter is used.